Jenkinsfile

```diff
@@ -64,7 +66,8 @@ pipeline {
         xvnc(useXauthority: true) {
           sh """
             ./full-build.sh --tp=${selectedTargetPlatform()} \
-              ${javaVersion() == 17 ? '' : '--toolchains releng/toolchains.xml -Pstrict-release-jdk'}
+              ${javaVersion() == 11 ? '--toolchains releng/toolchains.xml -Pstrict-jdk-11' : ''} \
```

Comment:
no j17 clause needed?

Reply:
I forgot that we wanted to always build with Java 11, I'll revert the check as it was before

Reply:
@cdietrich No wait: the check is correct like that: previously there were only Java 17 and Java 11, and if it was Java 17 we didn't do anything otherwise (Java 11) we set the toolchain (which corresponds to what I do now).
Now there's also Java 21, so we must explicitly check for Java 11.
For Java 17 nothing is required.